packages/cspell/samples/environment_config/load.txt

```diff
@@ -0,0 +1,93 @@
+                              [NEDERLANDS]
```

Comment:
Having a sample with a lot of Dutch will generate a lot of spelling issues if the Dutch dictionary isn't loaded.